Add symfony/webpack-encore-bundle (#630)
sonata-project/sandbox: CONTRIBUTING.md

Sonata Standard Edition
=======================

What's inside?
--------------

Sonata Standard Edition comes pre-configured with the following bundles:

> * Bundles from Symfony Standard distribution

Reviewer: We should avoid references to the term "distribution" since [distributions are no longer supported](https://symfony.com/distributions):
>Symfony Distributions are no longer used because they were superseded by Symfony 4, which creates micro applications by default and includes a tool called Symfony Flex to add new features in an automated way.
```suggestion
* Components and bundles from `symfony/*` packages
```

* Sonata Admin Bundles: Admin and Doctrine ORM Admin
* Sonata Ecommerce Bundles: Payment, Customer, Invoice, Order and Product
* Sonata Foundation Bundles: Notification, Formatter, Intl, Cache, Seo and Easy Extends
* Sonata Feature Bundles: Page, Media, News, User, Block, Timeline
* Api Bundles: FOSRestBundle, BazingaHateoasBundle, NelmioApiDocBundle and JMSSerializerBundle

Installation
------------

> ### Download sandbox files by one of possible examples

Reviewer: ```suggestion
### Download sandbox files by one of available methods
```

Curl:

    curl -L github https://github.com/sonata-project/sandbox-build/archive/master.tar.gz | tar xzv
    cd sandbox

Git:

>     git clone https://github.com/sonata-project/sandbox.git

Reviewer: ```suggestion
    git clone https://github.com/sonata-project/sandbox.git -b master --single-branch --depth=1
```

    cd sandbox
    git checkout master
    
### Prepare configuration

> * Copy configuration file: ``cp .env .env.local``

Reviewer: ```suggestion
* Copy the environment variables file: ``cp .env .env.local``
```

> * Edit ``.env.local`` to configure own environment

Reviewer: ```suggestion
* Edit ``.env.local`` to configure own settings
```

### Load fixtures data 

    vendor/bin/phing
    
* You should be ready to go ...

Vagrant Installation
--------------------

* vagrant up --provision --provider=virtualbox (Vagrant is going to get the environnement, install it for you and load sonata sample data)
* Configure your host ``sudo nano /etc/hosts`` and add this line ``192.168.33.99   sonata.local``
* Open your browser [here][link_sonata]

Run
---

> If you are running PHP 7.2 or above, you can use symfony to start the demo:

Reviewer: ```suggestion
If you are running PHP 7.2 or above, you can use the Symfony local web server to start the demo:
```
